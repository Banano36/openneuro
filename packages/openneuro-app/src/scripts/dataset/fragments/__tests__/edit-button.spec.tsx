import React from 'react'
import { render } from '@testing-library/react'
import { EditButton } from '../edit-button'

describe('EditButton component', () => {
  it('renders with default props', () => {
    const { asFragment } = render(<EditButton action={() => {}} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
