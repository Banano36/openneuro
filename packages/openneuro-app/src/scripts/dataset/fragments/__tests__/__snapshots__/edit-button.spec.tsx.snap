// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditButton component renders with default props 1`] = `
<DocumentFragment>
  <button
    aria-label="Edit"
    class="on-button on-button--medium on-button--default icon-text description-btn description-button-edit"
    role="button"
    type="button"
  >
    <i
      aria-hidden="true"
      class="fas fa-edit css-0"
    />
    Edit
  </button>
</DocumentFragment>
`;
